### DISCUSSION TITLE
Regridding MPASO while preserving horizontal resolution

### DISCUSSION THREAD

**Discussion by inciente** [E3SM-Project/E3SM#6792] [E3SM model help]
Hello all, 

I'm running WCYCL-1850 with the IcoswISC30E3r5 ocean grid. Where can I find all the map/remapping files related to that grid?

I've only been able to find files that remap onto the CMIP6 180x360 grid, but those decrease my horizontal resolution by quite a bit. 
I'd like to preserve as much of the horizontal resolution in my native grid as possible. What are my other options? 

Thank you

-- UPDATE
I found the regridding file '_map_IcoswISC30E3r5_to_0.5x0.5degree_bilinear.nc_' [here](https://web.lcrc.anl.gov/public/e3sm/diagnostics/mpas_analysis/maps/), but get the following error message when running ncremap with it: 

nco__open() unable to open file "map_IcoswISC30E3r5_to_0.5x0.5degree_bilinear.nc"
ERROR NC_EHDFERR Error at HDF5 layer
HINT: NC_EHDFERR errors indicate that the HDF5-backend to netCDF is unable to perform the requested task.

Any ideas on what might be wrong? I was able to remap my data using the _to_cmip6_180x360_ files just fine.

**Comment by jonbob** [comment]
That particular file looks like it's netcdf-4 format, which can be less portable. You can try converting your copy with "ncks -O -3 <filename>". But I can also point you to a scrip file for that grid if you are comfortable making your own mapping files. You can also look [here](https://web.lcrc.anl.gov/public/e3sm/inputdata/cpl/gridmaps/IcoswISC30E3r5/) for other mapping files we have made
